### DISCUSSION TITLE
How do I see server responses from individual run() commands

### DISCUSSION THREAD

**Discussion by db1996** [deployphp/deployer#2962] [Help needed]
Based on [this](https://deployer.org/docs/7.x/contrib/crontab) we made a task that creates all of our cron jobs for us. We have a txt file with a cron job on every line which it loops over.

This is our current task:

```
task('deploy:cronjobs', function () {
    run("echo '' | crontab -");
    $cron_file = file('cronjobs.txt');
    foreach ($cron_file as $line) {
        $line = trim(str_replace("\n", "", $line));
        run("(crontab -l; echo \"".$line."\") | crontab -");
    }
})->desc('Install the necessary cronjobs');
```

This seems to work without a problem in most of our projects however we're running into an issue in 1 of our projects where only 1 or 2 of the 4 cronjobs are actually deployed on the server. It's very inconsistent, sometimes it deploys all 4 cronjobs, but most of the time only 1 or 2 (without changing anything)

Is there any way to see if the `run("(crontab -l; echo \"".$line."\") | crontab -");` gives an error? to see if we're doing anything wrong

**Comment by antonmedv** [comment]
What about:
```php
$ct = run("crontab -l");
$ct .= $line;
run("echo $ct |  crontab -")
```

**Reply by db1996** [reply]
I can not get this solution to work, now none of the cronjobs are being deployed.
